cSpell.json

```diff
@@ -0,0 +1,84 @@
+{
+	"version": "0.2",
+	"language": "en",
+	"words": [
+		"Codegen",
+		"geolocation",
+		"iframes",
+		"navigations",
+		"pluggable",
+		"Polyfilling",
+		"Pytest",
+		"sandboxing",
+		"SIGHUP",
+		"SIGTERM",
+		"stringifiable"
+	],
+	"ignoreWords": [
+    "Boilerplates",
+		"Applitools",
+		"Blockstack",
+		"Checkly",
+		"Codecept",
+		"Instakittens",
+		"mockiavelli",
+		"Monocart",
+		"robotframework",
+		"Testim",
+		"Tesults",
+		"theheadless",
+		"whatwg"
+  ],
+	"patterns": [
+		{
+			"name": "Markdown links",
+			"pattern": "\\((.*)\\)",
+			"description": ""
+		},
+		{
+			"name": "Markdown code blocks",
+			"pattern": "/^(\\s*`{3,}).*[\\s\\S]*?^\\1/gmx",
+			"description": "Taken from the cSpell example at https://cspell.org/configuration/patterns/#verbose-regular-expressions"
+		},
+		{
+			"name": "Inline code blocks",
+			"pattern": "\\`([^\\`\\r\\n]+?)\\`",
+			"description": "https://stackoverflow.com/questions/41274241/how-to-capture-inline-markdown-code-but-not-a-markdown-code-fence-with-regex"
+		},
+		{
+			"name": "Link contents",
+			"pattern": "\\<a(.*)\\>",
+			"description": ""
+		},
+		{
+			"name": "Snippet references",
+			"pattern": "-- snippet:(.*)",
+			"description": ""
+		},
+		{
+			"name": "Snippet references 2",
+			"pattern": "\\<\\[sample:(.*)",
+			"description": "another kind of snippet reference"
+		},
+		{
+			"name": "Multi-line code blocks",
+			"pattern": "/^\\s*```[\\s\\S]*?^\\s*```/gm"
+		},
+		{
+			"name": "HTML Tags",
```

Comment:
ℹ️ I've adapted these patterns from various locations over time. `HTML Tags` is the only new one I added. It's pretty naive but got the job done in this case, catching every false positive reported issue.